addon/components/o-s-s/infinite-select.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="upf-infinte-select upf-floating-menu">
+  {{#if this.searchEnabled}}
+    {{#input-wrapper}}
+      <Input
+        value={{this._searchKeyword}} {{on "keyup" this.updateSearchKeyword}} type="text"
+        class="form-control upf-input" placeholder="Search..." />
```

Comment:
Shouldn't we put the placeholder either :
1) as a parameter, or 
2) in the en-us.yml ?